packages/compiler/src/experimental/typekit/kits/operation.ts

```diff
@@ -38,6 +40,11 @@ export interface OperationKit {
    * @param type type to check
    */
   is(type: Type): type is Operation;
+  /**
+   * Get the paging operation's metadata for an operation.
+   * @param operation operation to get the paging operation for
+   */
+  getPagingMetadata(operation: Operation): PagingOperation | undefined;
```

Comment:
Don't really like the mismatch between the type and the function name but we will be doing a typekit review for the final 1.0 to try to get to stable so might rename after

Reply:
Yes, I don't like this either, but I don't want to rename `PagingOperation` to `PagingMetadata` in my pr and the latter one is more proper i think.